zephyr/esp32/CMakeLists.txt

```diff
@@ -147,6 +147,7 @@ if(CONFIG_SOC_SERIES_ESP32)
     ../../components/spi_flash/spi_flash_chip_th.c
     ../../components/spi_flash/spi_flash_chip_winbond.c
     ../../components/spi_flash/spi_flash_os_func_noos.c
```

Comment:
Is the `spi_flash_os_func_noos.c` beeing used now?

Reply:
Yes, it is used.